zoom/utils.go

```diff
@@ -31,19 +38,31 @@ func connect(ctx context.Context, d *plugin.QueryData) (*zoom.Client, error) {
 	if zoomConfig.APISecret != nil {
 		apiSecret = *zoomConfig.APISecret
 	}
-
-	if apiKey == "" || apiSecret == "" {
-		// Credentials not set
-		return nil, errors.New("api_key and api_secret must be configured")
+	if zoomConfig.AccountID != nil {
+		accountID = *zoomConfig.AccountID
+	}
+	if zoomConfig.ClientID != nil {
+		clientID = *zoomConfig.ClientID
+	}
+	if zoomConfig.ClientSecret != nil {
+		clientSecret = *zoomConfig.ClientSecret
 	}
 
-	// Configure to automatically wait 1 sec between requests, per Zoom API requirements
-	conn := zoom.NewClient(apiKey, apiSecret)
-
-	// Save to cache
-	d.ConnectionManager.Cache.Set(cacheKey, conn)
+	if accountID == "" || clientID == "" || clientSecret == "" {
+		if apiKey == "" || apiSecret == "" {
```

Comment:
```suggestion
	if (accountID == "" || clientID == "" || clientSecret == "") && (apiKey == "" || apiSecret == "") {
```

I think we can consolidate this logic

Reply:
updated